scripts/zones/Balgas_Dais/npcs/Armoury_Crate.lua

```diff
@@ -602,6 +602,75 @@ local loot =
             {itemid =  4175, droprate =   99}, -- vile_elixir_+1
         },
     },
+
+    -- Come into my Parlor
```

Comment:
Looks like this was added to the wrong zone. This is Balgas Dias Armoury Crate and fight is in QuBia Arena